template.tpl

```diff
@@ -356,7 +362,8 @@ log({
 
 
 const initWidget = () => {
-  const productId = getProductId(data.widgetPid, null);
+  // default to data.widgetPidValue if it exists
+  const productId = data.widgetPidValue || getProductId(data.widgetPid, null);
```

Comment:
is there a need to also update the for events. as events also need to get the product id

Reply:
For events, shouldn't we always read `productId` from the event trigger? Or do customers have their own tag variable that evaluates to the correct `productId` during each ATC / Checkout event trigger as well?

From what I understand, we allow custom binding of `productId` for widget initialization because the variable will always be evaluated to the correct `productId` on the corresponding PDP page, but for events (especially Checkout event), the event trigger is not tied to PDP page

Reply:
Updated GTM template to include productId template variable for ATC event as well